alerter.cpp

```diff
@@ -27,6 +27,7 @@ int main() {
     alertInCelcius(400.5);
     alertInCelcius(303.6);
     std::cout << alertFailureCount << " alerts failed.\n";
+    assert(alertFailureCount != 0);

```

Comment:
This is a weak validation test. the failure count maybe !=0, but what if the expected value is 2 and return value is 1. The assert still passes!
The test case should exactly know how much is the expected value.